### PR TITLE
Openconnect: new options mtu

### PR DESCRIPTION
**Pull request by yousong**
Maintainer: @nmav 
Compile tested: ar71xx, LEDE-17.01
Run tested: run-tested

### CODE REVIEW

**Comment by yousong** [comment]
I plan to backport it to LEDE-17.01 branch if this got accepted.  What do you think?

File: net/openconnect/files/openconnect.sh [hunk 1]
```diff
@@ -6,6 +6,8 @@ init_proto "$@"
 proto_openconnect_init_config() {
 	proto_config_add_string "server"
 	proto_config_add_int "port"
+	proto_config_add_int "mtu"
+	proto_config_add_boolean "no_cert_check"
```
Comment (nmav) [inline net/openconnect/files/openconnect.sh]: Please remove that option. There is no point to have it.

File: net/openconnect/files/openconnect-wrapper [hunk 2]
```diff
@@ -10,32 +10,4 @@ test -z "$1" && exit 1
...
-{
-	if ! test -z "$pid";then
-		kill -2 $pid
-		wait $pid
-	fi
-	exit 0
-}
-
-trap cleanup2 2
-trap cleanup 1 3 6 15
```
Comment (nmav) [inline net/openconnect/files/openconnect-wrapper]: Why did you remove this?

File: net/openconnect/files/openconnect.sh (hunk 1 above)
Comment (yousong) [inline net/openconnect/files/openconnect.sh]: Okay, get it.  I am still using openconnect 7.06 on lede-17.01 branch.

File: net/openconnect/files/openconnect-wrapper (hunk 2 above)
Comment (yousong) [inline net/openconnect/files/openconnect-wrapper]: If there are no other routes other than those reported to netifd by vpnc-script with `proto_add_ipv4_route`, then netifd should take care of removing those routes on link down.

We also have `proto_kill_command` "$config" 2 in `proto_openconnect_teardown`.  This should also suffice

**Comment by yousong** [comment]
Hi, @nmav , are there any further comments on this one?

File: net/openconnect/files/openconnect-wrapper (hunk 2 above)
Comment (nmav) [inline net/openconnect/files/openconnect-wrapper]: Have you tested that?

Comment (nmav) [inline net/openconnect/files/openconnect-wrapper]: Let's separate the pull request to two different things. One that you add the mtu and this one. I remember there was a reason this wrapper was added and trapped all this signals; removing it would need good justification and testing, which is orthogonal to the mtu option.

Comment (yousong) [inline net/openconnect/files/openconnect-wrapper]: Yes, I am currently using the modified version, it works across ifdown and ifup.

The changes are already separated.  It contains three commits actually.

**Comment by nmav** [comment]
I'm still not convinced that the 3rd patch is right. Please open a separate PR with good argumentation on merging it. I've merged the other two to lede17 and master branches.
